containerd/main.go

```diff
@@ -48,7 +48,12 @@ var daemonFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "runtime,r",
 		Value: "runc",
-		Usage: "name of the OCI compliant runtime to use when executing containers",
+		Usage: "binary path of the OCI compliant runtime to use when executing containers",
```

Comment:
The previous comment was more accurate as it doesn't have to be a full path, as long as it is in the PATH.

Likewise for the usage of the new flag you added.